### PR TITLE
renames paranormal MULEbots to GHOULbots

### PR DESCRIPTION
**Pull request by ShizCalev**
:cl: ShizCalev
tweak: Renamed the "Paranormal MULEbot" to simply, "GHOULbot".
/:cl:

### CODE REVIEW

File: code/modules/mob/living/simple_animal/bot/mulebot.dm [hunk 1]
```diff
@@ -760,7 +760,7 @@
 		visible_message("<span class='notice'>[src] safeties are locked on.</span>")
 
 /mob/living/simple_animal/bot/mulebot/paranormal//allows ghosts only unless hacked to actually be useful
-	name = "paranormal MULEbot"
+	name = "\improper GHOULbot"
 	desc = "A Multiple Utility Load Effector bot. It only seems to accept paranormal forces, and for this reason is fucking useless."
```
Comment (Fikou) [inline code/modules/mob/living/simple_animal/bot/mulebot.dm:764]: it still says Multiple Utility Load Effector bot in the desc

**Comment by tralezab** [comment]
oh my god i love you shiz

File: code/modules/mob/living/simple_animal/bot/mulebot.dm (hunk 1 above)
Comment (ShizCalev) [reply code/modules/mob/living/simple_animal/bot/mulebot.dm:764]: intended. feel free to suggest a change though!

Comment (ATH1909) [reply code/modules/mob/living/simple_animal/bot/mulebot.dm:764]: Ghastly Hauling Of UnLife bot? Eh, it's a bit of a stretch, and do ss13 ghosts really count as unlife, seeing as how they're more "dead" than "undead"?

**Comment by IndieanaJones** [comment]
Speed merge now

**Pull request merged**